src/tests/end-to-end/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+#bigobj support needed for windows compilation
+if(MSVC)
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj")
```

Comment:
Maybe use of 
`set_property(TARGET tests-simple-study PROPERTY /bigobj)`
or something like that would be more specific

Reply:
I used the option defined in `solver/CMakeList.txt`. This seems to be the way of adding bigobj support in Windows.